cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -127,6 +165,44 @@ public Service generateService() {
                 Collections.singletonList(createServicePort(REST_API_PORT_NAME, REST_API_PORT, REST_API_PORT, "TCP")));
     }
 
+    public ConfigMap generateMetricsConfigMap() {
+        if (isMetricsEnabled()) {
+            Map<String, String> data = new HashMap<>();
```

Comment:
I wonder if we could use something like `Collections.singletonMap(...)` here instead than creating the Map.